test/it/positive.spec.js

```diff
@@ -1,13 +1,59 @@
+const tmp = require("tmp");
+const fs = require("fs-extra");
 const compiler = require("../utils/compiler");
 const listSubDirectories = require("../utils/listSubDirectories");
+const { getTsConfigByContext } = require("../utils/tsconfig");
+const {
+  getTestContext,
+  handleWidget,
+  handlePageElementsMap,
+  createDeclarationsFilesByConfigPath
+} = require("../utils/typesByContext");
+
 const POSITIVE_ROOT_PATH = "test/it/code-samples/positive";
 const positiveRoots = listSubDirectories(POSITIVE_ROOT_PATH);
 
-describe("typescript - positive scenarios", () => {
+describe("typescript - positive scenarios - configPaths flow", () => {
+  it.each(positiveRoots)(
+    "should successfully compile %s folder",
+    async tsRootPath => {
+      const testTmpDirPath = tmp.dirSync().name;
```

Comment:
try to put the "environment preparations" into a dedicated function, so it'll be easy to understand that we run the same tests with the 2 environments